hw3/src/pages/list.jsx

```diff
@@ -0,0 +1,64 @@
+import Navbar from "../components/navbar";
+import { useState, useEffect } from "react";
+
+const Header = () => {
+  return (
+    <div className="text-center mt-10">
+      <p className="text-5xl font-bold font-sans text-blue-600">
+        The List of Countries
+      </p>
+    </div>
+  );
+};
+
+const GetCountries = () => {
+  const [countries, setCountries] = useState([]);
+
+  useEffect(() => {
+    const fetchCountries = async () => {
+      try {
+        const response = await fetch(
+          "https://cs464p564-frontend-api.vercel.app/api/countries"
+        );
+        if (!response.ok) {
+          throw new Error(`HTTP error! status: ${response.status}`);
+        }
+        const data = await response.json();
+        setCountries(data);
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      }
+    };
```

Comment:
Since this is a function it will not and should not do harm if placed outside, with that you could just return the value then save it using use-state or variable function setter